apps/extension/src/final/extension-popup/components/trading-copilot-dialogs/trading-copilot-dialogs.tsx

```diff
@@ -0,0 +1,166 @@
+import { Icon as IdrissIcon } from '@idriss-xyz/ui/icon';
+
+import {Dialog, Icon, LazyImage} from "shared/ui";
+import {useCommandQuery} from "shared/messaging";
+import {GetEnsInfoCommand} from "application/trading-copilot";
+
+const exampleTradingCopilotDialogs = [
+  {
+    details: {
+      name: 'vitalik.eth',
+      amount: 150,
+      crypto: 'ETH',
+      when: '15 mins ago',
+      title: "Trading copilot",
+    },
+    user: {
+      balance: 200,
+      crypto: 'ETH',
+    },
+    id: 'dialog1',
+  },
+  {
+    details: {
+      name: 'vitalikk.eth',
+      amount: 300,
+      crypto: 'ETH',
+      when: '30 mins ago',
+      title: "Trading copilot",
+    },
+    user: {
+      balance: 200,
+      crypto: 'ETH',
+    },
+    id: 'dialog2',
+  },
+];
+
+type TradingCopilotDialogsProperties = {
+  activeDialogId: string | null,
+  closeDialog: () => void,
+}
+
+export const TradingCopilotDialogs = ({ activeDialogId, closeDialog }: TradingCopilotDialogsProperties) => {
+  const activeDialog = exampleTradingCopilotDialogs.find((dialog) => {return dialog.id === activeDialogId});
+
+  if (!activeDialog) return null;
+
+  return (
+    <>
+      {
+        exampleTradingCopilotDialogs.map((dialog, index) => {
+          return (
+            <TradingCopilotDialog key={index} dialog={dialog} activeDialogId={activeDialogId} closeDialog={closeDialog} />
+          )
+        })
+      }
+    </>
+  )
+};
+
+type TradingCopilotFormValues = {
+  amount: string;
+};
+
+const EMPTY_FORM: TradingCopilotFormValues = {
+  amount: '',
+};
+
+type TradingCopilotDialogProperties = {
+  dialog: {
+    details: {
+      name: string,
+      amount: number,
+      crypto: string,
+      when: string,
+      title: string,
+    }
+    user: {
+      balance: number,
+      crypto: string,
+    },
+    id: string,
+  },
+  activeDialogId: string | null,
+  closeDialog: () => void
+}
+
+const TradingCopilotDialog = ({dialog: {details, user, id}, activeDialogId, closeDialog}: TradingCopilotDialogProperties) => {
+  const form = useForm<TradingCopilotFormValues>({
+    defaultValues: EMPTY_FORM,
+  });
+
+  const avatarQuery = useCommandQuery({
+    command: new GetEnsInfoCommand({
+      ensName: details.name,
+      infoKey: 'avatar',
+    }),
+    staleTime: Number.POSITIVE_INFINITY,
+  });
+
+  return (
+    <Dialog open={id === activeDialogId} closeDialog={closeDialog} title={details.title}>
+      <div className="flex flex-col gap-y-6">
+        <div className='flex flex-row gap-3'>
+          <LazyImage
+            src={avatarQuery.data}
+            className="size-12 rounded-full"
+            fallbackComponent={
+              <Icon size={48} name="PersonIcon" className="rounded-full"/>
+            }
+          />
+          <div className="flex flex-col gap-2">
+            <p className="text-heading5 text-neutralGreen-900">{details.name} <span className="text-neutral-700 font-normal">purchased {details.amount} {details.crypto}</span></p>
```

Comment:
we should have `text-heading4` for the {details.name} paragraph, and `text-neutral-600 text-body-3` for the purchased... span to match the designs